c_emulator/riscv_sim.c

```diff
@@ -287,17 +287,17 @@ static int process_args(int argc, char **argv)
       rv_enable_misaligned = true;
       break;
     case OPT_PMP_COUNT:
-      pmp_count = atol(optarg);
-      fprintf(stderr, "PMP count: %lld\n", pmp_count);
+      pmp_count = (uint8_t)atoi(optarg);
+      fprintf(stderr, "PMP count: %u\n", pmp_count);
```

Comment:
```suggestion
      fprintf(stderr, "PMP count: %" PRId64 "\n", pmp_count);
```

No need to change the types, just use the ugly 64-bit format macros instead?